components/ActivityIndicator/README.md

```diff
@@ -86,88 +69,157 @@ import MaterialComponents
+let activityIndicator = MDCActivityIndicator()
+activityIndicator.sizeToFit()
 activityIndicator.indicatorMode = .determinate
 activityIndicator.progress = 0.5
 view.addSubview(activityIndicator)
 
-// Start animation
+// Show the indicator
 activityIndicator.startAnimating()
-...
-// Stop animation
+
+// Hide the indicator
 activityIndicator.stopAnimating()
 ```
 
 #### Objective-C
 
 ``` objc
-MDCActivityIndicator *activityIndicator =
-    [[MDCActivityIndicator alloc] initWithFrame:CGRectMake(0, 0, 32, 32)];
+MDCActivityIndicator *activityIndicator = [[MDCActivityIndicator alloc] init];
+[activityIndicator sizeToFit];
 activityIndicator.indicatorMode = MDCActivityIndicatorModeDeterminate;
 activityIndicator.progress = 0.5;
 [view addSubview:activityIndicator];
 
-// Start animation
+// Show the indicator
 [activityIndicator startAnimating];
-...
-// Stop animation
+
+// Hide the indicator
 [activityIndicator stopAnimating];
 ```
 <!--</div>-->
 
-- - -
+### How to theme an activity indicator
 
-## Related Components
+You must first add the Color Themer extension to your project:
 
-<ul class="icon-list">
-  <li class="icon-list-item icon-list-item--components"><a href="../ProgressView">Progress Views</a></li>
-</ul>
+``` bash
+pod 'MaterialComponents/ActivityIndicator+Extensions/ColorThemer'
+```
+
+You can then import the theming APIs:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+import MaterialComponents.MaterialActivityIndicator_ColorThemer
+```
+
+#### Objective-C
+
+``` objc
+#import "MaterialActivityIndicator+ColorThemer.h"
+```
+<!--</div>-->
+
+MDCActivityIndicatorColorThemer allows you to theme an activity indicator with your app's color
+scheme.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+let colorScheme: MDCSemanticColorScheme()
+
+let activityIndicator = MDCActivityIndicator()
+MDCActivityIndicatorColorThemer.apply(colorScheme, to: activityIndicator)
+```
+
+#### Objective-C
+
+``` objc
+id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] init];
+
+MDCActivityIndicator *activityIndicator = [[MDCActivityIndicator alloc] init];
+[MDCActivityIndicatorColorThemer applySemanticColorScheme:colorScheme
+                                      toActivityIndicator:activityIndicator];
+```
+<!--</div>-->
+
+### How to set multiple indeterminate colors
+
+Indeterminate activity indicators support showing multiple colors via the `cycleColors` API.
+Consider using this property if your brand consists of more than one primary color.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+let activityIndicator = MDCActivityIndicator()
+activityIndicator.cycleColors = [.blue, .red, .green, .yellow]
+```
+
+#### Objective-C
+
+``` objc
+MDCActivityIndicator *activityIndicator = [[MDCActivityIndicator alloc] init];
+activityIndicator.cycleColors =  @[ [UIColor blueColor],
```

Comment:
Please use property syntax for colors. e.g., `UIColor.blueColor`.

Reply:
Done.